views/layout.hbs

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>IronBeers</title>
+</head>
+<body>
+    <nav>
+        <ul>
+            <li><a href="/"></a>HOME</li>
```

Comment:
There is a small typo here which makes the link unclickable. The `</a>` tag should be after HOME.